.github/ISSUE_TEMPLATE/general-question.yml

```
@@ -25,29 +22,14 @@ body:
     attributes:
       label: Code/Examples
       description: >-
-        Please post any supporting code around your question here.
-
-  - type: textarea
-    attributes:
-      label: Other information
-      description: >-
-        Please provide any attempts on answering the question, or any related information you already found here.
+        Please post any supporting code around your question here. Don't forget to include the Python version and package version, if it's relevant.
```

```suggestion
        Please post any supporting code around your question here. 
```

Personally I agree with your original removal of this 🙂